README.md

```diff
@@ -10,6 +10,7 @@ for Stanford and Cornell Sinopia-to-ILS/LSP workflows.
 Based on the documentation, [Running Airflow in Docker](https://airflow.apache.org/docs/apache-airflow/stable/start/docker.html):
 
 1. Clone repository `git clone https://github.com/LD4P/ils-middleware`
+1. Uncomment the line `# - ./dags:/opt/airflow/dags` in docker-compose.yaml
```

Comment:
i think i just pulled the latest `main`, and i see this line, but it's not commented out by default for me.  maybe just tacking on ` (if it isn't already)` and dropping `# ` would help?